decidim-assemblies/spec/permissions/decidim/assemblies/permissions_spec.rb

```diff
@@ -431,7 +431,12 @@
       it_behaves_like "allows any action on subject", :assembly
       it_behaves_like "allows any action on subject", :assembly_member
       it_behaves_like "allows any action on subject", :assembly_user_role
-      it_behaves_like "allows any action on subject", :space_private_user
+      context "when private assembly" do
```

Comment:
```suggestion

      context "when private assembly" do
```

Reply:
fixed in [bb74aa7](https://github.com/decidim/decidim/pull/12031/commits/bb74aa7c35909f58337992441e4040338a5c8537)